content/en/blog/_posts/2022-12-06-kubernetes-1.26-blog.md

```diff
@@ -0,0 +1,210 @@
+
+### Progress on the CSI migration process
+
+Following the GA of the [core Container Storage Interface (CSI) Migration](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/625-csi-migration) feature in the previous release, CSI migration is an on-going effort that we've been working on for a few releases now, and this release continues to add (and remove) features aligned with the migration's goals.
+
+#### _Delegate FSGroup to CSI Driver_ graduated to stable
+
+This feature allows Kubernetes to [supply the pod's `fsGroup` to the CSI driver when a volume is mounted](https://github.com/kubernetes/enhancements/issues/2317) so that the driver can utilize mount options to control volume permissions.
+Previously, the kubelet would always apply the `fsGroup`ownership and permission change to files in the volume according to the policy specified  in the Pod's `.spec.securityContext.fsGroupChangePolicy` field.
+Starting with this release, CSI drivers have the option to apply the `fsGroup` settings during attach or mount time of the volumes.
+
+This feature allows Kubernetes to [supply the pod's `FSGroup` to the CSI driver when a volume is mounted](https://github.com/kubernetes/enhancements/issues/2317) so that the driver can utilize mount options to control volume permissions. Previously, the kubelet would always apply the `fsGroup`ownership and permission change to files in the volume according to the policy specified  in the `Pod.spec.securityContext.fsGroupChangePolicy` field. Starting with this release, CSI drivers have the option to apply the `fsGroup` settings during attach or mount time of the volumes.
+
+#### CSI migration for Azure File and vSphere graduated to stable
+
+Both the [vSphere](https://github.com/kubernetes/enhancements/issues/1491) and [Azure](https://github.com/kubernetes/enhancements/issues/1885) in-tree driver migration to CSI have graduated to Stable. You can find more information about them in the [vSphere CSI driver](https://github.com/kubernetes-sigs/vsphere-csi-driver) and [Azure File CSI driver](https://github.com/kubernetes-sigs/azurefile-csi-driver) repositories.
+
+#### In-tree GlusterFS driver removal
+
+Already deprecated in the v1.25 release, the in-tree GlusterFS driver was [removed](https://github.com/kubernetes/enhancements/issues/3446) in this release.
+
+#### In-tree OpenStack Cinder driver removal
+
+This release removed the deprecated in-tree storage integration for OpenStack (the `cinder` volume type). You should migrate to external cloud provider and CSI driver from https://github.com/kubernetes/cloud-provider-openstack instead. For more information, visit [Cinder in-tree to CSI driver migration](https://github.com/kubernetes/enhancements/issues/1489).
+
```

Comment:
we can leave it in if we identified this as major theme. I was just wondering if we wanted to make it more concise and since this is already mentioned in this blog separately (at the bottom of the list) & in the Deprecation blog I guess no information will be lost.

Reply:
Will keep them for now, but leave this unresolved for a couple of days more so we can take a new look with other changes in place.